Tests/Adyen Tests/Analytics/SyncAnalyticsEventDataSourceTests.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import XCTest
+@_spi(AdyenInternal) @testable import Adyen
+
+final class SyncAnalyticsEventDataSourceTests: XCTestCase {
```

Comment:
Is this still needed?

Reply:
Let's add this class whenever we have tests?